test/src/test/resources/rmq-proxy-home/conf/logback_proxy.xml

```diff
@@ -289,10 +289,10 @@
 
     <appender name="RocketmqCommercialAppender"
               class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${user.home}/logs/rocketmqlogs/${brokerLogDir}/commercial.log</file>
+        <file>${user.home}/logs/rocketmqlogs/${brokerLogDir:-${file.separator}}/commercial.log</file>
```

Comment:
Why not replace `/` with `${file.separator}` here?

Reply:
Fixed, please re-review